engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -264,6 +308,18 @@ public void update(float delta) {
         }
     }
 
+    /**
+     Called whenever there's a change to WorldConfigurator properties
+     Provides information about the changed property as well as old and new value in {@link PropertyChangeEvent}
+     */
+    @Override
+    public void propertyChange(PropertyChangeEvent propertyChangeEvent) {
+        System.out.println(propertyChangeEvent.getPropertyName());
+        if (propertyChangeEvent.getPropertyName() == "Trees") {
```

Comment:
Checkstyle makes a good point here:
> ERROR:
> Literal Strings should be compared using equals(), not '=='.

```suggestion
        if (propertyChangeEvent.getPropertyName().equals("Trees")) {
```

Reference equality rarely works as you would expect.

Reply:
Also, why is this checking specifically for changes to the "Tree" property in particular?

Reply:
That was test code for the WorldConfigurator change.
I forgot to remove it after that PR got merged.